provider/cmd/pulumi-resource-nuage/nuage_provider/container_function.py

```diff
@@ -168,12 +202,47 @@ def __init__(self, name: str, args: ContainerFunctionArgs, props: Optional[dict]
             )
         )
         
+        # FIXME: Thhis doesn't work because pulumi Command doesn't have option to run on update right now 
+        # As it work only on create, it keeps having the same tag on updates
+        #untag_command = local.Command(
+        #    f"untag-{name}-image",
+        #    create=repository_url.apply(
+        #        lambda repository_url: f"docker rmi {repository_url}"
+        #    ), 
+        #    opts=pulumi.ResourceOptions(depends_on=[image])
+        #)
+        
         # Define inline policies for role definition
+        log_group = aws.cloudwatch.LogGroup(
+            f"{name}-log-group", 
+            name=f"/aws/lambda/{name}",
```

Comment:
I suggest you use the function name instead of the resource name, as the actual function will potentially have a suffix and the log group would mismatch